frontend/lib/src/components/widgets/DataFrame/hooks/useColumnSort.test.ts

```diff
@@ -222,4 +222,82 @@ describe("useColumnSort hook", () => {
       Array.from(sortedDataDesc).sort().reverse()
     )
   })
+
+  it("should respect explicit direction parameter when sorting", () => {
+    const { result } = renderHook(() =>
+      useColumnSort(
+        MOCK_PROPS.numRows,
+        MOCK_PROPS.columns,
+        MOCK_PROPS.getCellContent
+      )
+    )
+    const SELECTED_COLUMN = 0
+
+    // Sort explicitly descending
+    act(() => {
+      const { sortColumn } = result.current
+      sortColumn?.(MOCK_COLUMNS[SELECTED_COLUMN].indexNumber, "desc")
+    })
+
+    // Column header should contain descending sort icon
+    expect(result.current.columns[SELECTED_COLUMN].title).toContain("↓")
+
+    const sortedDataDesc = []
+    for (let i = 0; i < MOCK_PROPS.numRows; i++) {
+      sortedDataDesc.push(
+        (result.current.getCellContent([SELECTED_COLUMN, i]) as NumberCell)
+          .data
+      )
+    }
+
+    // Verify data is sorted in descending order
```

Comment:
nit: I would look into nesting this test in another `describe` with two `it`s in it, like

```typescript
describe("should respect explicit direction parameter when sorting", () => {
  it("descending", () => {})
  it("ascending", () => {})
})
```

Reply:
Ok, I will split up into two tests 👍